server/controllers/users/userController.ts

```diff
@@ -0,0 +1,78 @@
+import { Response } from 'express';
+import Interviewee from '../../database/Models/Interviewee';
+import { RequestType } from '../../utils';
+import validateQuery from '../../utils/validation/queryValidation';
+
+const getAllReviews = async (req: RequestType, res: Response) => {
+  const { page = '1', saved = undefined } = req.query;
```

Comment:
If we don't send a saved then it undefined, so you don't need add it

Reply:
Yes but then I would have to only access it using req.query.saved each time I need to use it, so this is much simpler